basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -38,7 +41,19 @@ public static int average (int[] array) {
     }
 
     public static int lowestAverage(int[][] array) {
-        
+        int[] averageArray = new int[array.length];
+        for ( int i  = 0; i < array.length; i++) {
+            int total = 0;
+            int average = 0;
+
+            for (int j = 0; j < array[i].length; j++){
+                total += array[i][j];
+            }
+            average = total/array[i].length;
+            averageArray[i] = average;
+        }
+        Arrays.sort(averageArray);
+        return averageArray[0];
```

Comment:
This is just returning what the average was, not the array that produced that average.